canonical-webteam/global-nav
# Add Prettier; Use embedded SVG for logo

## Done

- Added Prettier
- Replaced uncompressed logo pulled from assets server with embedded SVG code

## QA

- Sanity check code
- Check logo displays correctly as per this codepen: https://codepen.io/barrymcgee/pen/exvobj


Fixes #70
